### DISCUSSION TITLE
How can we ensure accessibility from the start?

### DISCUSSION THREAD

**Discussion by mrmetaverse** [metaversityfoundation/metaversity#18] [Q&A]
Rather than back-filling for accessibility, how can we ensure that what we put out there is accessible from the beginning?

**Comment by apolancojr** [comment]
It's important to engage not only accessibility specialists, but also users with special needs as well. This is an important topic to me and is related to my Doctor of Design dissertation - [Developing Best Practices for Designing Digital Assessments for Middle-school aged Deaf or Hard of Hearing Students in the United States](https://www.researchgate.net/publication/354632709_Developing_Best_Practices_for_Designing_Digital_Assessments_for_Middle-school_aged_Deaf_or_Hard_of_Hearing_Students_in_the_United_States)
